tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "include": [
+      "dist/*.js"
```

Comment:
do built files contain jsdoc annotations? should this be `lib/**/*.js`?

Reply:
They do.

1. `npm run build`
1. `less dist/samsam.js`
1. Observe JSDoc annotations